### ISSUE TITLE
Upload the latest release to PyPi Package index

### ISSUE THREAD

**Issue by jdanekrh** [iwanbk/nyamuk#19]
Page https://pypi.python.org/pypi/nyamuk lists v0.1.1 as the latest (and only) release. There was v0.2.0 released in May 2016. I'd like to be able to install it with pip.

**Comment by gbour** [comment]
Hi @jdanekrh 

@iwanbk did not had time to do it. Now that I am maintainer of the package on pypi, I'll do it quickly
 (and also fix #20)

**Comment by gbour** [comment]
I just uploaded 0.2.0 release on pypi

```
$ pip install nyamuk
Collecting nyamuk
Installing collected packages: nyamuk
Successfully installed nyamuk-0.2.0
```

Please add other comments to this tickets if you encounter other install issues
